### PR TITLE
Chore: Evaluate SearchMessagesView - TypeScript

### PR DESCRIPTION
**Pull request by reinaldonetof**
## How to test or reproduce

- Go to a channel
- Then, click on `Search` icon
- Check the messages from search message
- Also, test with an encryption channel 

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves a current function)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Documentation update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [x] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat.ReactNative)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/views/SearchMessagesView/index.tsx [hunk 1]
```diff
@@ -272,6 +273,7 @@ class SearchMessagesView extends React.Component<ISearchMessagesViewProps, ISear
 				theme={theme}
 				onPress={() => this.jumpToMessage({ item })}
 				jumpToMessage={() => this.jumpToMessage({ item })}
+				rid={item.rid}
```
Comment (GleidsonDaniel) [inline app/views/SearchMessagesView/index.tsx:276]:
```suggestion
				rid={message.rid}
```
If the item is a message, this should also be the message shouldn't it? Just to follow the pattern haha

File: app/views/SearchMessagesView/index.tsx [hunk 2]
```diff
@@ -254,16 +256,15 @@ class SearchMessagesView extends React.Component<ISearchMessagesViewProps, ISear
...
 		const { user, baseUrl, theme, useRealName } = this.props;
 		return (
 			<Message
-				// @ts-ignore IMessage | TMessageModel?
-				item={item}
+				item={message}
 				baseUrl={baseUrl}
 				user={user}
 				timeFormat='MMM Do YYYY, h:mm:ss a'
-				isHeader
```
Comment (GleidsonDaniel) [inline app/views/SearchMessagesView/index.tsx:266]: Why did you delete this property?

**Review by GleidsonDaniel — CHANGES_REQUESTED** [review]

File: app/views/SearchMessagesView/index.tsx (hunk 1 above)
Comment (reinaldonetof) [reply app/views/SearchMessagesView/index.tsx:276]: haha yeap, will be the same! But you are right 😁

File: app/views/SearchMessagesView/index.tsx (hunk 2 above)
Comment (reinaldonetof) [reply app/views/SearchMessagesView/index.tsx:266]: the property isHeader doesn't exist anymore! I searched where this was modified and was modified when Diego added a get isHeader in `app/containers/message/index.tsx`. What do you think?

https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/880/files#diff-2655a2c9c415ee7ce1aa809d9e64ff6bda856982c95a6ae7bb17dce5e4853135R123

**Review by GleidsonDaniel — APPROVED** [review]
